wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/rapidreactcommandbot/subsystems/Intake.java

```diff
@@ -0,0 +1,31 @@
+package edu.wpi.first.wpilibj.examples.rapidreactcommandbot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.motorcontrol.PWMSparkMax;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import static edu.wpi.first.wpilibj.examples.rapidreactcommandbot.Constants.IntakeConstants;
+
+public class Intake extends SubsystemBase {
+  private final PWMSparkMax m_motor = new PWMSparkMax(IntakeConstants.kMotorPort);
+  private final DoubleSolenoid m_pistons =
+      new DoubleSolenoid(PneumaticsModuleType.REVPH,
+                         IntakeConstants.kSolenoidPorts[1],
+                         IntakeConstants.kSolenoidPorts[2]);
+
+  public Command deployCommand() {
+    return runOnce(() -> {
```

Comment:
This would mean that the subsystem can be idle while the motor is running. Not good practice.

Reply:
I think this is user preference, but we can change this to a runcommand I guess